website/docs/r/workers_kv_namespace.html.markdown

```diff
@@ -0,0 +1,36 @@
+---
+layout: "cloudflare"
+page_title: "Cloudflare: cloudflare_workers_kv_namespace"
+sidebar_current: "docs-cloudflare-resource-workers-kv-namespace"
+description: |-
+  Provides the ability to manage Cloudflare Workers KV Namespace features.
+---
+
+# cloudflare_workers_kv_namespace
+
+Provides a Workers KV Namespace
+
+## Example Usage
+
+```hcl
+resource "cloudflare_workers_kv_namespace" "example" {
+  title        = "test-namespace"
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `title` - (Required) The name of the namespace you wish to create
+
+
+## Import
+
+KV Namespace settings can be imported using the it's title e.g.
```

Comment:
I think we should be using IDs here as they will be unique whereas title values are not guaranteed. What do you think?

Reply:
Namespace titles are unique per Cloudflare account so I thought it may be simpler to just use its title rather than ID.

Reply:
That's fair enough. Have you considered use cases where a title contains spaces? How would we import them using this method?

Reply:
Good point will update it shortly.